docs/getting-started/dockstore-workflows.rst

```diff
@@ -114,31 +114,25 @@ This is the newest way of getting content onto Dockstore and is by far the most
 GitHub Apps, Dockstore can react to changes on GitHub as they are made, keeping Dockstore synced
 with GitHub automatically.
 
-To install our GitHub App on either a repository or organization,
-navigate to the ``/my-workflows`` page and click add workflow. Follow the steps for GitHub Apps and
-you will be redirected to GitHub where you can select which repositories to install the Dockstore
-GitHub app on. The process is shown in the following images.
+Installing the GitHub App is simply -- navigate to ``/my-workflows``, click the ``+`` button on the left hand sidebar, select ``Register using GitHub Apps``, and then click
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can grant the app access to specific repositories within whatever organization you are installing into.
```

Comment:
Nit-picky and you can leave it, but I think, for example, github.com/aofarrel is a user, not an org. And you make that distinction elsewhere.

Reply:
I struggled with wording it concisely. "You'll then be redirected to GitHub where you can grant the app access to specific repositories within whatever organization or user you are installing into." is one option, but I'm wondering if that might confuse people as it may imply you can be a member of a user. Also, iirc, we are already encouraging people to have their workflows on GitHub organizations rather than personal repos (I may be confusing this with advice regarding Dockstore organizations and collections though).

Reply:
I think you can leave it as you have it then. I did say "nit-picky" :)

Reply:
I strive to remove all nits, but this one seems resistant to the metaphorical pick.